lib/react_client/bridge.dart

```diff
@@ -116,13 +119,16 @@ class Component2BridgeImpl extends Component2Bridge {
         String componentName,
         String location,
         String propFullName,
-        // This is a required argument of PropTypes but is usally hidden from the JS consumer.
+        // This is a required argument of PropTypes but is usually hidden from the JS consumer.
         String secret,
       ) {
         // Create a Dart consumable version of the JsMap.
         var convertedProps = JsBackedMap.fromJs(props);
         // Call the users validator with the newly wrapped props.
-        var error = validator(convertedProps, propName, componentName, location, propFullName);
+        var error = validator(
+            convertedProps,
+            PropValidatorInfo(
+                propName: propName, componentName: componentName, location: location, propFullName: propFullName));
```

Comment:
#nit trailing commas might be nice here for formatting purposes
```suggestion
                propName: propName, componentName: componentName, location: location, propFullName: propFullName,));
```